test/environmentCreate/SpecificationCreate.test.tsx

```diff
@@ -33,6 +35,27 @@ describe("<SpecificationCreate />", () => {
     const switchButton = component.getByLabelText("YAML", { exact: false });
     fireEvent.click(switchButton);
 
+    expect(
+      screen.getByText((content, element) => {
+        return (
+          element?.textContent === "channels:  -dependencies:  -variables: {}"
+        );
+      })
+    );
+
+    act(() => {
+      store.dispatch(requestedPackagesChanged(["python>5.0", "numpy"]));
+    });
+
+    expect(
+      screen.getByText((content, element) => {
+        return (
+          element?.textContent ===
+          "channels: []dependencies:  - python>5.0  - numpyvariables: {}"
+        );
+      })
+    );
```

Comment:
Please move this into its own test.

Reply:
Context for this: we discussed how the tests should match their description. The problem here is that I inserted code that does more than just checking that switching to YAML works. So I need to move this new logic into a standalone test (but copy-pasting the current code is fine).